docs/content/commands.md

```diff
@@ -13,3 +13,13 @@
 ## XML DSL
 
 <p align="center"><img src="../images/camelRouteXMLDSL.gif" alt="Camel Route XML DSL command" class="zoom" width="90%"/></p>
+
+# Create a Camel Quarkus project (using Camel JBang export)
```

Comment:
I think it is worthy to mention also in list of features here https://github.com/camel-tooling/camel-lsp-client-vscode/blob/main/docs/index.md?plain=1#L30 and link to commands.md

Reply:
provided